src/ComposerRequireChecker/FileLocator/LocateFilesByGlobPattern.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace ComposerRequireChecker\FileLocator;
+
+
+use Webmozart\Glob\Glob;
+
+class LocateFilesByGlobPattern
+{
+
+    public function __invoke(array $globPatterns = [], $rootDir) : \Generator
```

Comment:
Return type is probably `iterable` here (highest possible abstraction), but needs a specific `@return` with a detailed type